sbt_common/display/src/test/scala/uk/ac/wellcome/display/models/DisplayJacksonModuleTestBase.scala

```diff
@@ -0,0 +1,139 @@
+package uk.ac.wellcome.display.models
+
+import io.circe.Json
+import io.circe.parser._
+import org.scalatest.Suite
+import uk.ac.wellcome.models._
+
+trait DisplayJacksonModuleTestBase {this: Suite =>
+
+  def items(its: List[IdentifiedItem]) =
+    its
+      .map { it =>
+        s"""{
+          "id": "${it.canonicalId}",
+          "type": "${it.ontologyType}",
+          "locations": [
+            ${locations(it.locations)}
+          ]
+        }"""
+      }
+      .mkString(",")
+
+  def locations(locations: List[Location]) =
+    locations
+      .map { location(_) }
+      .mkString(",")
+
+  def location(loc: Location) = loc match {
+    case l: DigitalLocation => digitalLocation(l)
+    case l: PhysicalLocation => physicalLocation(l)
+  }
+
+  def digitalLocation(loc: DigitalLocation) =
+    s"""{
+      "type": "${loc.ontologyType}",
+      "locationType": "${loc.locationType}",
+      "url": "${loc.url}",
+      "license": ${license(loc.license)}
+    }"""
+
+  def physicalLocation(loc: PhysicalLocation) =
+    s"""
+       {
+        "type": "${loc.ontologyType}",
+        "locationType": "${loc.locationType}",
+        "label": "${loc.label}"
+       }
+     """
+
+  def license(license: License) =
+    s"""{
+      "label": "${license.label}",
+      "licenseType": "${license.licenseType}",
+      "type": "${license.ontologyType}",
+      "url": "${license.url}"
+    }"""
+
+  def identifier(identifier: SourceIdentifier) =
+    s"""{
+      "type": "Identifier",
+      "identifierScheme": "${identifier.identifierScheme}",
+      "value": "${identifier.value}"
+    }"""
+
+  def identifiedOrUnidentifiable[T](displayableAgent: Displayable[T],
```

Comment:
What does this do? A comment would be appreciated, because it’s not obvious to me.